.github/workflows/publish-docs.yml

```
@@ -12,7 +12,9 @@ jobs:
   update-gh-pages:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v1
+    - uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
```

```suggestion
    - uses: actions/checkout@v4
      with:
        fetch-depth: 0
        ref: gh-pages
```
WDYT? To be safe that we are not the right branch?

---

The script we run comes from the ref that was pushed. It does not exist in gh-pages

---

ah right.